test/integration/provider/admin/dashboards_controller_test.rb

```diff
@@ -29,4 +29,12 @@ class Provider::Admin::DashboardsControllerTest < ActionDispatch::IntegrationTes
     get provider_admin_dashboard_path
     assert_xpath(xpath_selector, element_text)
   end
+
+  test '#update_current_user_after_login called once' do
+    Admin::Api::BaseController.expects(:update_current_user_after_login).once
```

Comment:
Probably this:
```suggestion
    Admin::Api::BaseController.any_instance.expects(:update_current_user_after_login).once
```

Reply:
I can't make it fail! Without skipping the method, it is called 3 times according to the server logs
- Provider::Admin::DashboardsController
- Provider::Admin::Dashboard::PotentialUpgradesController
- Provider::Admin::Dashboard::NewAccountsController

However, the test only satisfies the following when added in setup, before login:
```ruby
Provider::Admin::DashboardsController.any_instance.expects(:update_current_user_after_login).times(2)
```

And this is always true, with or without skipping:
```ruby
Provider::Admin::Dashboard::PotentialUpgradesController.any_instance.expects(:update_current_user_after_login).once
Provider::Admin::Dashboard::NewAccountsController.any_instance.expects(:update_current_user_after_login).once
```